projects/kit/abstract/input-slider.ts

```diff
@@ -136,6 +165,10 @@ export abstract class AbstractTuiInputSlider<T>
         return this.length / this.computedSteps;
     }
 
+    get computedKeySteps(): TuiKeySteps {
+        return [[0, this.min], ...(this.keySteps || []), [100, this.max]];
```

Comment:
I'm not sure about this getter, because for each call to change detection a new object will be created

Reply:
maybe it's better

```ts
get computedKeySteps(): TuiKeySteps {
   return this.computePureKeySteps(this.keySteps, this.min, this.max);
}

@tuiPure
private computePureKeySteps(keySteps, min, max) {
 //...
}
```

Reply:
✅